.github/workflows/cross-version-tests.yml

```diff
@@ -40,6 +40,7 @@ defaults:
 jobs:
   set-matrix:
     runs-on: ubuntu-latest
+    timeout-minutes: 60
```

Comment:
Most of cross-version tests finish within 5 ~ 10 minutes, but `spark-dev / models` test takes 45 ~ 50.

https://github.com/mlflow/mlflow/actions/runs/3321267943/jobs/5488725035

Reply:
The default is 360 minutes (6 hours).